Ar_MethodHw.js

```diff
@@ -0,0 +1,185 @@
+// // 1.
+//
+Array.prototype.myForEach = function(callback) {
+  for (let i = 0; i < this.length; i++) {
+    callback(this[i]);
+  }
+};
+
+//
+// //[1, 2, 3].myForEach((el) => {
+//   //console.log(el)
+// //})
+// // 2.
+
+// })
+// SOLUTION USING A LOOP INSTEAD
+// Array.prototype.myMap = function(callback) {
+//  let result =[];
+//  for(let i = 0; i < this.length; i++) {
+//    result.push(callback(this[i]))
+//
+//  }
+//  return result
+//  console.log(result)
+// };
+//
+// Array.prototype.myMap = function(anyT){
+//    let result = [];
+//    this.myForEach(element => {
+//
+//      result.push(anyT(element));
+//    })
+//
+//    return result;
+// }
+//
+//
+// let double = [1, 2, 3].myMap((el) => {
+//  return el * 2;
+// })
+// console.log(double);
+// 3.
+// Array.prototype.myFilter = function(callback){
+//     let result = [];
+//     this.myForEach(element => {
+//
+//       result.push(callback(element));
+//     })
+//
+//     return result;
+//   }
+//
+// let ages = [32, 33, 16, 40].myFilter((el)=>{
+//   return el >= 18;
+// })
+// console.log(ages)
+//
+// let ages = [32, 33, 16, 40].myFilter((el)=>{
+//     if ( el >= 18){
+//       console.log(el)
+//     }else{
+//      (el <= 18)
+//     }
+//     console.log()
+//   return el;
+// })
+// console.log(ages)
+
+// 4. The every() method checks if all elements in an array pass a test (provided as a function).
+//
+// The every() method executes the function once for each element present in the array:
+//
+// If it finds an array element where the function returns a false value, every() returns false (and does not check the remaining values)
+// If no false occur, every() returns true
+// Note: every() does not execute the function for array elements without values.
+//
+// Note: every() does not change the original array
+//4
+Array.prototype.myEvery = function(callback) {
```

Comment:
I feel like you know what myEvery is supposed to return (a boolean) but you're going to return `true` every time here - right? So, what is myEvery supposed to check for?